app/controllers/merchant_invoices_controller.rb

```diff
@@ -2,4 +2,9 @@ class MerchantInvoicesController < ApplicationController
   def index
     @merchant = Merchant.find(params[:merchant_id])
   end
+
+  def show 
+    @invoice = Invoice.find(params[:invoice_id])
+    @customer = Customer.find(@invoice.customer_id)
```

Comment:
John I noticed that I dont think this @customer instance variable is being used for anything.  I wonder if you initially went that route and just forgot to remove it because I noticed the tests still pass without it.  Regardless, the code looks nice and works as it should